substrait_consumer/context.py

```diff
@@ -2,10 +2,8 @@
 
 import jpype.imports
 
-import substrait_consumer.java_definitions as java
```

Comment:
Seems odd to move jmoorts down, is it because you want to be able to only include them conditionally?

Reply:
yeah, that's exactly it...otherwise we need to install java on some of the docker containers that don't need it.